sail/CLI/Entry.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+namespace SailCMS\CLI;
+
+use League\Flysystem\FilesystemException;
+use SailCMS\Errors\ACLException;
+use SailCMS\Errors\DatabaseException;
+use SailCMS\Errors\EntryException;
+use SailCMS\Errors\PermissionException;
+use SailCMS\Models\Entry as EntryModel;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Entry extends Command
+{
+    protected static $defaultDescription = 'Indexation of entries';
+    protected static $defaultName = 'entry:index';
```

Comment:
maybe change this for entries:index or run:indexing

Reply:
OK ! I'll change it.